src/v2/models/AppState.js

```diff
@@ -105,9 +105,33 @@ export default Model.extend({
   },
 
   shouldReRenderView (transformedResponse) {
+    _.mixin({
```

Comment:
create a folder under V2 called `mixins` and add a file `mixins.js` and add this function their and use it here.